test/unit-test/FreeRTOS_ARP/FreeRTOS_ARP_utest.c

```diff
@@ -0,0 +1,947 @@
+/* Include Unity header */
+#include "unity.h"
+
+/* Include standard libraries */
+#include <stdlib.h>
+#include <string.h>
+#include <stdint.h>
+
+#include "mock_FreeRTOS_IP.h"
+#include "mock_FreeRTOS_IP_Private.h"
+#include "mock_task.h"
+#include "mock_NetworkBufferManagement.h"
+
+#include "FreeRTOS_ARP.h"
+
+#include "FreeRTOS_ARP_stubs.c"
+#include "catch_assert.h"
+
+#include "FreeRTOSIPConfig.h"
+
+#define ARPCacheEntryToCheck    2
+
+#if ARPCacheEntryToCheck >= ipconfigARP_CACHE_ENTRIES
+    #error "ARPCacheEntryToCheck cannot be greater than ipconfigARP_CACHE_ENTRIES"
+#endif
+
+extern ARPCacheRow_t xARPCache[ ipconfigARP_CACHE_ENTRIES ];
+
+void FillARPCache( void )
```

Comment:
static void

Reply:
This is not required, so removed the function too.